Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
-name = "poi-crosschecker"
+name = "poi-radio"
 version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-graphcast_sdk = { package = "graphcast-sdk", git = "https://github.com/graphops/graphcast-sdk", rev="ee94148" }
+graphcast_sdk = { package = "graphcast-sdk", git = "https://github.com/axiomatic-aardvark/graphcast-sdk" }
```

Comment:
iirc you can reference with actual repo commit with 'rev' even if the commit hasn't been merging? Is there something preventing that development process such that a fork is needed?

Reply:
You're right, I was trying out a bunch of experimental stuff before that and that's why I had a fork, but I'll look into how to use a local git dependency in the future, that'd be easiest. Also yeah if a commit has been pushed (but not merged to main) we can still reference it, so no need of forks really

Reply:
reminder to the link before you merge, thanks!